src/explore-education-statistics-frontend/src/modules/data-catalogue/DataCataloguePage.tsx

```diff
@@ -112,11 +113,24 @@ const DataCataloguePage: NextPage<Props> = ({
   const handleDownloadFormSubmit: DownloadFormSubmitHandler = async ({
     files,
   }) => {
-    if (!state.query.release) {
+    const { release } = state.query;
+
+    if (!release) {
       throw new Error('Release has not been selected');
     }
 
-    await downloadService.downloadFiles(state.query.release.id, files);
+    await downloadService.downloadFiles(release.id, files);
+
+    const availableFiles = state.subjects.map(subject => subject.file);
+    const selectedFilenames = files
+      .map(fileId => availableFiles.find(file => file.id === fileId)?.fileName)
+      .join(', ');
+
+    logEvent({
+      category: 'Downloads',
+      action: 'Data Catalogue page selected files download',
+      label: `Publication: ${state.query.publication?.title}, Release: ${release.title} File: ${selectedFilenames}`,
```

Comment:
I guess we don't want to change the label value prefix for the filenames from `File: ` to `Files: ` here because it might upset filtering on historical data already in GA?